src/Kunstmaan/TranslatorBundle/Command/ImportTranslationsFromFileCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Kunstmaan\TranslatorBundle\Command;
+
+use Box\Spout\Common\Type;
+use Box\Spout\Reader\ReaderFactory;
+use Box\Spout\Reader\XLSX\Sheet;
+use Doctrine\ORM\EntityManager;
+use Kunstmaan\MediaBundle\Helper\ExtensionGuesserFactory;
+use Kunstmaan\TranslatorBundle\Entity\Translation;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesser;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class CalculateCoordinatesCommand.
+ */
+class ImportTranslationsFromFileCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('kuma:translator:import-file')
+            ->setDescription('Export file with translations, supported formats are xlsx, ods, csv')
+            ->addArgument('file', InputArgument::REQUIRED, 'The full path of the file');
+    }
+
+    /**
+     * @throws LogicException
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|null|void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $file = $input->getArgument('file');
+        $filePath = realpath(dirname($file)).DIRECTORY_SEPARATOR;
+        $fileName = basename($file);
+        $file = $filePath.$fileName;
+
+        if (!file_exists($file)) {
+            throw new LogicException(sprintf('Can not find file in %s', $file));
+        }
+
+        $format = (new File($file))->guessExtension();
+        if(!Type::XLSX == $format && !Type::CSV == $format && !Type::ODS == $format) {
```

Comment:
Maybe `inarray` instead of multiple ands?

Reply:
Thanks for taking the time to go through my PR! I appreciate it very much. We are this week going to be spending a lot of time trying to get most of the pull requests reviewed and we will also be brainstorming about this PR if we want to include it into the UI or keep it as a command only. What do you think?

Reply:
I think better choice, is UI because is more flexible. For us as administrators there's no difference if we import data by command or import button. Sometimes people responsible for completion text on pages want to faster change and import is good option.

From the other side why don't move the logic to separately service and give users both options?